<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="userLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="codeLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="registerBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="confirmLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sendcodeBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>发送验证码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
